dashboard/src/components/AppNew/AppNew.tsx

```diff
@@ -200,6 +243,18 @@ class AppNew extends React.Component<IAppNewProps, IAppNewState> {
   public handleReleaseNameChange = (e: React.FormEvent<HTMLInputElement>) => {
     this.setState({ releaseName: e.currentTarget.value });
   };
+  public handleChartVersionChange = (e: React.FormEvent<HTMLSelectElement>) => {
+    const { versions } = this.props.selected;
+    const cv = versions.find(v => v.attributes.version === e.currentTarget.value);
```

Comment:
I think this could be made simpler, looks like a lot of work to just get the `repoName` and `chartName`. We should be able to from the URL from `this.props.chartID` and `e.currentTarget.value` alone.